package.json

```
@@ -132,4 +134,4 @@
   "//TODO": [
     "For @patternfly/react-icons: find another solution for icons from older react-icons version and remove this dependency."
   ]
-}
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)